models/playlist.js

```diff
@@ -1,17 +1,55 @@
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../knexfile')[environment];
+const database = require('knex')(configuration);
+
 class Playlist {
 
-  constructor(playlistData, favoriteData) {
+  constructor(playlistData) {
     this.id = playlistData.id;
     this.title = playlistData.title;
     this.songCount = 0;
     this.songAvgRating = 0;
-    this.favorites = [];
+    this.favorites;
+    this.created_at = playlistData.created_at;
+    this.updated_at = playlistData.updated_at;
+  }
+
+ async getFavorite(favoriteId) {
```

Comment:
Thank you for building these helper functions, this will help with refactoring, in the future.